client/packages/executor/src/executionManager/sideEffect.ts

```diff
@@ -405,6 +435,14 @@ export class SideEffect extends EventEmitter {
      * @param bidAmount Amount of the incoming bid
      */
     processBid(signer: string, bidAmount: number) {
+        // Add the executor bid to the list 
+        this.biddingEngine.storeWhoBidOnWhat(this.id, signer);
+        // TODO: should this be here or in "generateBid"?
+        // Add how much it bid
+        this.lastBids.push(bidAmount);
+        // Add the xtx's bid to its history
+        this.storeXtxBidHistory(signer, bidAmount);
```

Comment:
rename also

Reply:
Done! ✌️
I removed the method there because it's already present in the bidding engine.